src/components/AnimatedNavbar.tsx

```diff
@@ -9,23 +9,28 @@ const AnimatedNavbar = ({
   OverflowHeaderComponent,
   TopNavbarComponent,
   headerHeight,
+  headerBackgroundColor,
+  borderBottomColor
 }: AnimatedNavbarProps) => {
   const [headerOpacity, overflowHeaderOpacity] = useAnimateNavbar(
     scroll,
     imageHeight,
-    headerHeight
+    headerHeight,
   );
+  const colorScheme = useColorScheme();
 
   return (
     <>
       <Animated.View
         style={[
           styles.container,
-          styles.header,
           {
             zIndex: headerOpacity,
             height: headerHeight,
             opacity: headerOpacity,
+            backgroundColor: headerBackgroundColor = colorScheme == 'light' ? '#fff' : '#121212',
+            borderBottomWidth: StyleSheet.hairlineWidth,
+            borderBottomColor: borderBottomColor = colorScheme == 'light' ? 'rgb(216, 216, 216)' : '#222',
```

Comment:
Better to use hex codes instead of RGB to be consistent with the rest of the styling.
```suggestion
            borderBottomColor: borderBottomColor = colorScheme == 'light' ? '#D8D8D8' : '#222',
```